scripts/liminal

```diff
@@ -84,9 +84,14 @@ def deploy_liminal_core_internal(clean):
 def docker_compose_command(command_name, args):
     docker_compose_path, project_dir = get_docker_compose_paths()
     concated_args = ' '.join(args)
-    subprocess.call(
+    if 'follow' in str(args):
+        subprocess.call(
+                [f'docker-compose -f "{docker_compose_path}" -p liminal --project-directory {project_dir} {command_name} {concated_args}'],
+                env=os.environ, shell=True)
+    output = subprocess.run(
```

Comment:
better to prepare the command string once, and decide if to invoke it with subprocess.call or run.